xapian-bindings/configure.ac

```diff
@@ -131,6 +169,10 @@ dnl CXXFLAGS for the duration of configure.
 save_CXXFLAGS=$CXXFLAGS
 CXXFLAGS="$CXXFLAGS $XAPIAN_CXXFLAGS"
 
+if test yes = "$enable_letor"; then
+    CXXFLAGS="$CXXFLAGS $XAPIANLETOR_CXXFLAGS"
+fi
```

Comment:
Looks like you didn't fix the overindentation here.

Reply:
fixed in  a fixup commit.